addons/acr/units/CfgUnits.hpp

```diff
@@ -1,6 +1,5 @@
 //Units
-class UKSF_ACR_Rifleman: UKSF_ACR_Soldier_Base
-{
+class UKSF_ACR_Rifleman: UKSF_ACR_Soldier_Baseb {
```

Comment:
`...Baseb` typo?

Reply:
think I've not pushed something here, doesn't exist my side